MyBlog.API/Controllers/UserController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MyBlog.Infrastructure.Entities;
+using MyBlog.Service.Areas.Users;
+
+namespace MyBlog.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class UserTestController : ControllerBase
+{
+    private readonly ILogger<UserTestController> _logger;
+    private readonly IUserService _userService;
+    
+    public UserTestController(ILogger<UserTestController> logger, IUserService userService)
+    {
+        _logger = logger;
+        _userService = userService;
+    }
+
+    /// <summary>
+    /// Get list of Users
+    /// </summary>
+    /// <returns>Returns User list</returns>
+    /// <response code="200">Success</response>
+    [HttpGet]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    public async Task<IActionResult> GetListAsync()
+    {
+        var users = await _userService.GetListAsync();
+        
+        return Ok(users);
+    }
```

Comment:
your return statement: 
return Ok(users)
so, you return List<User>
then, you should edit your ProducesResponseType tag:
[ProducesResponseType(typeof(List< User >), StatusCodes.Status200OK)]

Apply this rule for all endpoints in your controller